config.guess

```diff
@@ -991,6 +991,9 @@ EOF
     m68*:Linux:*:*)
 	echo "$UNAME_MACHINE"-unknown-linux-"$LIBC"
 	exit ;;
+    loongarch32:Linux:*:* | loongarch64:Linux:*:*)
+        echo "$UNAME_MACHINE"-unknown-linux-"$LIBC"
+        exit ;;
```

Comment:
Please update the `config.{guess,sub}` files in a separate commit; and please just copy [the latest version from upstream](https://git.savannah.gnu.org/gitweb/?p=config.git;a=summary) instead of patching only what you need, because this pair of files are meant to be vendored/used this way.